src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -981,33 +988,34 @@ It is also possible to change the ProActive Nodes Account using the 'Change Acco
 
 When you click on 'GUI Edit', the following window appears:
 
-image::config_editor_general.png[align="center"]
+image::config_editor_general.png[align=center, width=612, height=306]
 
 In the general tab, the user can specify:
 
 * The *ProActive Scheduler* location.
-* The *JRE* location (usually something like `C:\Program Files\Java\jdk1.6.0_12`).
+* The *JRE* location (usually something like `C:\Program Files\ProActive Agent\schedworker\jre`).
 * The *numbers of Runtimes and Nodes* (the number of spawned processes and the number of ProActive Nodes per process).
+* The *numbers of workers* (the number of ProActive Nodes allocated in the Compute Host)
 * The *JVM options*. Note that if the parameter contains `${rank}`, it will be dynamically replaced by the ProActive Node rank starting from 0.
 * The *On Runtime Exit* script. A script executed after a ProActive Node exits. This can be useful to perform additional cleaning operation. Note that the script receives as parameter the PID of the ProActive Node.
 * The user can set a *memory limit* that will prevent the spawned processes to exceed a specified amount of RAM. If a spawned process or its child process requires more memory, it will be killed as well as its child processes. Note that this limit is disabled by default (0 means no limit) and a ProActive Node will require at least 128 MBytes.
 * It is possible to list all available *network interfaces* by clicking on the "Refresh" button and add the selected network interface name as a value of the `proactive.net.interface` property by clicking on "Use" button. See the ProActive documentation for further information.
-* The user can specify the *protocol* (PNP or PAMR) to be used by the ProActive Node for incoming communications.
+* The user can specify the *protocol* (PNP or PAMR) to be used by the ProActive Node for incoming communications. In case the selected protocol is PAMR, the user has to add `-Dproactive.pamr.router.address` JVM property to specify the router address which corresponds generally to the server hostname.
```

Comment:
- If the selection protocol is PAMR, ...
- the user has to add `-Dproactive.pamr.router.address` JVM parameter / the user has to configure the `proactive.pamr.router.address` JVM property